### DISCUSSION TITLE
How to reinitialise ACE?

### DISCUSSION THREAD

**Discussion by DonzoBundestag** [acemod/ACE3#8500] [Q&A]
Hello i have a system on my server where players can change the side but also have to re initialized. So all my ace settings don't get applied on the person switching sides.
my question is: how can reinitialize ACE via a script command?

**Comment by jonpas** [comment]
What does "reinitialize" mean?

**Reply by DonzoBundestag** [reply]
Oh so i talked with the dev who made the switch system. he said ". You just have to add a extra line that initialise ACE but ask them which one is it". So sorry i meant initialise

**Reply by DonzoBundestag** [reply]
So is there a way to let ACE Fetch the current mission settings again?

**Reply by jonpas** [reply]
I fail to see why you would need to do anything. There is no reason current mission settings would not be applied with a side change.

What actually breaks? What happens and what do you expect to happen as a user?

**Reply by DonzoBundestag** [reply]
OK, so im on a Altis life Framework. The problem is that when a player changes his side ingame via a Duty Script, ace dose not behave like it sould (Mission settings). Fatal injuries happen, player dies instant and so on. But if the Person switches back via the system to civ ace works just fine. So i thought that maybe the Settings dont get transferd but now i know that when i reinitialise the client, ace dose not work (calls the core\init.sqf again) so its not the switch of side causing it directly but the init.sqf. The thing is evry clinet dose call the init.sqf on joining the server. so i thought that maybe after the swich ace dose not keep up or somthing.

**Reply by DonzoBundestag** [reply]
so i tgought mybe its possible to init ace again to make it work

**Reply by jonpas** [reply]
There is no such thing as manually initializing ACE3. Your scripts definitely break something, we can't help without seeing them.

**Reply by DonzoBundestag** [reply]
Would it help you if post the file here?

**Reply by jonpas** [reply]
Sure.

**Comment by DonzoBundestag** [comment]
@jonpas is there an eventHandler that is executed when the player connects to the server? That would help me a lot and here are the tow files getting executed:
Init.sqf in the core Folder: https://pastebin.com/j43QXWgc
medic init: https://pastebin.com/1cdfkjVh

**Reply by jonpas** [reply]
Not from ACE3.

I don't see anything wrong there, but I don't use Life and we can't support it from our side.

**Reply by DonzoBundestag** [reply]
@jonpas is there an eventHandler that is executed when the player connects to the server?

**Reply by jonpas** [reply]
From vanilla yes, from ACE3 no.

I suggest you jump on ACE3 Slack, this is becoming too much of a discussion.
